File: pinyougou_parent/pinyougou_seckill_interface/src/main/java/com/pinyougou/seckill/service/SeckillService.java
package com.pinyougou.seckill.service;

import com.pinyougou.pojo.TbSeckillGoods;

import java.util.List;

public interface SeckillService {
    /**
     * 查询秒杀商品列表
     */
    public List<TbSeckillGoods> findSeckillGoodsList();

    /**
     * 基于秒杀商品id查询秒杀商品
     */
    public TbSeckillGoods findOne(Long seckillGoodsId);

    /**
     * 秒杀下单功能
     */
    public void saveOrder(Long seckillGoodsId,String userId);
}

File: pinyougou_parent/pinyougou_seckill_web/src/main/webapp/js/service/seckillService.js
//服务层
app.service('seckillService',function($http){
	    	
	//查询秒杀商品列表
	this.findSeckillGoodsList=function(){
		return $http.get('seckill/findSeckillGoodsList.do');
	}

	//查询秒杀商品列表
	this.findOne=function (seckillGoodsId) {
		return $http.get("seckill/findOne.do?seckillGoodsId="+seckillGoodsId);
    }

    //秒杀下单
	this.saveOrder=function (seckillGoodsId) {
        return $http.get("seckill/saveOrder.do?seckillGoodsId="+seckillGoodsId);
    }
});
